editions/tw5.com/tiddlers/widgets/ButtonWidget.tid

```diff
@@ -38,7 +38,7 @@ The content of the `<$button>` widget is displayed within the button.
 |popup |Title of a state tiddler for a popup that is toggled when the button is clicked. See PopupMechanism for details |
 |popupTitle |Title of a state tiddler for a popup that is toggled when the button is clicked. In difference to the <<.attr popup>> attribute, ''no'' TextReference is used. See PopupMechanism for details |
 |popupAbsCoords |<<.from-version "5.2.4">> If set to ''yes'' writes absolute coordinates to the tiddler referenced by the <<.attr popup>>. If set to ''no'' (the default) uses relative coordinates. See [[Coordinate Systems]] for details |
-|aria-label |Optional [[Accessibility]] label |
+|aria-label |Optional <<.dlink-ex Accessibility to:https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA>> label |
```

Comment:
I think there should be a tiddler `Accessibility` tagged: `Definitions` with some info and the external link. We should keep users on the page

Reply:
Agreed, though I'm not sure I'm up to creating an `Accessibility in Tiddlywiki` page, as I feel there should be more information there than I can provide.

Reply:
I think it would be OK to create the tiddler, tag it and then simply add

```
To learn more about accessibility visit: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA
```

So the link is not hidden behind an alias. So users know what they click on. @Jermolene What do you think?